src/main/java/org/snomed/snowstorm/core/data/services/identifier/LocalSequentialIdentifierSource.java

```diff
@@ -86,7 +86,7 @@ private int findHighestIdentifierSequence(int namespaceId, String partitionId) {
 				componentClass = Relationship.class;
 				idField = Relationship.Fields.RELATIONSHIP_ID;
 				break;
-			case "06":
+			case "16":
```

Comment:
@kaicode Shouldn't it be "06" or "16" here? I can see POSTCOORDINATED_EXPRESSION_PARTITION_ID  is still set to "06".